src/Core/Configuration/OptionsConsumer.cs

```diff
@@ -1,44 +1,9 @@
-﻿using Microsoft.Extensions.Primitives;
-
-using System;
-
-namespace CodeMonkeys.Configuration
+﻿namespace CodeMonkeys.Configuration
 {
-    public abstract class OptionsConsumer<TOptions> : IDisposable
-        where TOptions : Options
+    public abstract class OptionsConsumer<TOptions>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Convert this 'abstract' class to a concrete class with a protected constructor.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)